commit 7f787704cdcd55df18d204c5251cdd6a3cedd604
Author: Sage Weil <sweil@redhat.com>
Date:   Tue May 29 16:47:07 2018 -0500

    mgr: use entity_addrvec_t for MgrMap
    
    This does several entity_addr_t -> entity_addrvec_t conversions across the
    mgr code.
    
    (Note that entity_addr_t and entity_addrvec_t already encode in an
    interoperable way.)
    
    Signed-off-by: Sage Weil <sage@redhat.com>

diff --git a/src/mgr/MgrStandby.cc b/src/mgr/MgrStandby.cc
index 5e267feab69..d6c24ef97a1 100644
--- a/src/mgr/MgrStandby.cc
+++ b/src/mgr/MgrStandby.cc
@@ -199,7 +199,7 @@ void MgrStandby::send_beacon()
   // as available in the map)
   bool available = active_mgr != nullptr && active_mgr->is_initialized();
 
-  auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
+  auto addrs = available ? active_mgr->get_server_addrs() : entity_addrvec_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id() << dendl;
 
   map<string,string> metadata;
@@ -209,7 +209,7 @@ void MgrStandby::send_beacon()
   MMgrBeacon *m = new MMgrBeacon(monc.get_fsid(),
 				 monc.get_global_id(),
                                  g_conf->name.get_id(),
-                                 addr,
+                                 addrs,
                                  available,
 				 std::move(module_info),
 				 std::move(metadata));
